packages/gasket-plugin-cypress/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "@gasket/plugin-cypress",
+  "version": "1.0.0",
+  "description": "Integrates Cypress based testing into your Gasket application",
+  "main": "lib",
+  "scripts": {
+    "lint": "eslint .",
+    "lint:fix": "npm run lint -- --fix",
+    "test": "jest",
+    "test:watch": "jest --watchAll",
+    "test:coverage": "jest --coverage",
+    "start:local": "gasket start --env local",
+    "cypress": "cypress open",
+    "cypress:headless": "cypress run",
+    "e2e": "start-server-and-test start:local http://localhost:8080 cypress",
+    "e2e:headless": "start-server-and-test start:local http://localhost:8080 cypress:headless"
```

Comment:
You do not need these scripts in the plugin's package.json. They are not intended to be run from here, correct?
```suggestion
```